bindings/python/dlite-python.i

```diff
@@ -1296,4 +1296,12 @@ PyObject *_get_DLiteError(void);
 
 %pythoncode %{
   DLiteError = _dlite._get_DLiteError()
+  
+  class DLiteStorageBase:
```

Comment:
These should be done properly, as done for DLiteError above - fixes issue #182

Reply:
I pushed an implementation for proper access to the underlying C objects for DLiteStorageBase and DLiteMappingBase to PR #208